docs/okeps/okep-5085-localnet-api.md

```diff
@@ -0,0 +1,595 @@
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| Role                | Select the network role in the pod.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | No       |
+| PhysicalNetworkName | The OVN bridge mapping network name is configured on the node.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | No       |
+| MTU                 | The maximum transmission unit (MTU).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Yes      |
```

Comment:
@ormergi If MTU is optional, is there a default value that is used?

Reply:
The default MTU for localnet topology should be 1500 https://github.com/ormergi/ovn-kubernetes/blob/5f86091e9bc7e787d171041c6242e4dcc7a81269/docs/okeps/okep-5085-localnet-api.md#mtu